crates/ruma-federation-api/src/knock/create_knock_event_template/v1.rs

```diff
@@ -0,0 +1,43 @@
+//! [GET /_matrix/federation/v1/make_knock/{roomId}/{userId}](https://github.com/matrix-org/matrix-doc/blob/master/proposals/2403-knock.md#get-_matrixfederationv1make_knockroomiduserid)
+
+use ruma_api::ruma_api;
+use ruma_events::pdu::Pdu;
+use ruma_identifiers::{RoomId, RoomVersionId, UserId};
+use ruma_serde::Raw;
+
+ruma_api! {
+    metadata: {
+        description: "Send a request for a knock event template to a resident server.",
+        name: "create_knock_event_template",
+        method: GET,
+        path: "/_matrix/federation/v1/make_knock/:room_id/:user_id",
+        rate_limited: false,
+        authentication: ServerSignatures,
+    }
+
+    request : {
```

Comment:
Inconsistent formatting

```suggestion
    request: {
```